### DISCUSSION TITLE
Heltec LoRa 32 V3 support status?

### DISCUSSION THREAD

**Discussion by pa1wbu** [lora-aprs/LoRa_APRS_iGate#274] [Q&A]
hi there,

Today the postman brought me a Heltec LoRa 32 in a V3 version. Sofar my impression is that that revision hardware is not (or not yet maybe?) supported by the code base?

Can someone maybe confirm or deny my assumption?

73, Wilko

**Comment by peterus** [comment]
Hi Wilko!

The board is currently not supported as it has a different CPU architecture.
But I have already a board lying around and trying to find some time to check everything ;)

73, Peter

**Comment by pa1wbu** [comment]
hi Peter,

More than happy to test it once you feel things are 'good enough'. 

73, Wilko
